src/script/components/panel/PanelActions.tsx

```diff
@@ -37,10 +37,10 @@ export interface PanelActionsProps {
 const PanelActions: React.FC<PanelActionsProps> = ({items}) => {
   return (
     <Fragment>
-      {items.map(({click, identifier, icon, label, size = {}}) => (
+      {items.map(({click, identifier, icon, label, size}) => (
```

Comment:
How about this default: `size = {width: 16, height: 16}`?

Reply:
But when you set only `width`, `height` will not be set.